### PR TITLE
feat: check duplication of an entry based on its attributes

### PR DESCRIPTION
**Pull request by navid-kalaei**
implementation of PR #261

### CODE REVIEW

File: src/ports/web/api/duplicates.rs [hunk 1]
```diff
@@ -22,7 +23,13 @@ pub fn post_duplicates(
         let db = db.shared()?;
         db.all_places()?
     };
-    let results = usecases::find_duplicate_place(&new_place, &possible_duplicate_entries);
+
+    let connection = db.exclusive()?;
+    let target_place = create_place(&*connection, new_place, None)?;
```
Comment (flosse) [inline src/ports/web/api/duplicates.rs]: Don't create a new place!
Compare a potential new place with existing ones.

**Comment by flosse** [comment]
The test fails:
```sh
failures:
    ports::web::api::tests::check_place_duplicate
```
Pleas have a look at my [introduction](https://github.com/kartevonmorgen/openfairdb/pull/261#issue-421369755).

**Comment by flosse** [comment]
@navid-kalaei Thank's for your effort. Unfortunately your solution is a bit complicated. It could be much easier (see updated PR #261).

**Comment by navid-kalaei** [comment]
Hey @flosse , I sincerely appreciate you for showing me a better way of thinking and opening up the new ideas of seeing development through the window of simplicity and efficiency. I carefully went through every single line of code you have provided and got a better feeling about the OpenFairDB and Rust. You have taught me by the actual implementation of how a change of view leads to a better software development. Once again, thank you for your piece of art.

**Comment by flosse** [comment]
@navid-kalaei Thanks for your understanding.
And I want to mention that most of the elegant work was done by @enoui :trophy:

**Pull request closed without merging**